### PR TITLE
remove SCSS file and move font folder

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?
- Finish MUI update! 
- Remove `scss` files (font.scss, global.scss, theme.scss, viewport.scss) 🧹 🧹🧹
- Move font from `src `to `public` folder for convenience.
- closes: https://github.com/saddle-finance/saddle-frontend/issues/782

## Context
https://github.com/saddle-finance/saddle-frontend/issues/782

### CODE REVIEW

**Review by ug02fast — APPROVED** [review]

File: public/fonts/index.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+@font-face {
+  font-family: "Camphor";
+  font-weight: 700;
+  font-style: normal;
+  src: url("./Camphor-Bold.eot"); /* IE9 Compat Modes */
+  src: url("./Camphor-Bold.eot?#iefix") format("embedded-opentype"),
+    /* IE6-IE8 */ url("./Camphor-Bold.woff") format("woff"),
+    /* Modern Browsers */ url("./Camphor-Bold.ttf") format("truetype"),
+    /* Safari, Android, iOS */
+      url("./Camphor-Bold.svg#e37ef20b6d73827754050f31838d5c3f") format("svg"); /* Legacy iOS */
```
Comment (hammeiam) [inline public/fonts/index.css:64]: what is this hash?

**Review by hammeiam — APPROVED** [review]
Congratulations on finishing this monumental task!!

File: public/fonts/index.css (hunk 1 above)
Comment (nkrulikovsky) [reply public/fonts/index.css:64]: This font css was from the old `SCSS` file and and I believe that hash is required for `svg font` to work on iPhone or iPad. https://web.archive.org/web/20150102141618/http://www.fluorcow.com/embedding-fonts-on-a-website

Comment (ug02fast) [reply public/fonts/index.css:64]: I looked into it a little too. Seems like it's still necessary to keep that identifier.

**Pull request merged**